packages/react-storage/src/components/StorageImage/types.ts

```diff
@@ -7,25 +7,25 @@ export interface StorageImageProps extends Omit<ImageProps, 'src'> {
   // Note: a new Storage.get request is made only when the imgKey gets updated after the initial
   /**
    * @deprecated
-   * `imgKey` will be replaced with `path` in the next major version of Amplify UI
+   * `imgKey` will be replaced with `path` in the next major version of Amplify UI. See https://ui.docs.amplify.aws/react/connected-components/storage/storageimage#props
```

Comment:
Let's align to the message structure used in `HAS_DEPRECATED_PROPS_MESSAGE`, while _most likely_ we will remove in the next MV there is always a chance we won't, and Amplify UI covers the entire monorepo, not just the react-storage package 😅 
```suggestion
   * `imgKey` will be replaced with `path` in a future major version. See https://ui.docs.amplify.aws/react/connected-components/storage/storageimage#props
```